data_structures/linked_list/singly_linked_list.py

```diff
@@ -13,13 +16,13 @@ def insert_tail(self, data):
             self.insert_head(data)  # If this is first node, call insert_head
         else:
             temp = self.head
-            while temp.next != None:  # traverse to last node
+            while temp.next is not None:  # traverse to last node
```

Comment:
Even more Pythonic to just do __while temp.next:__

And below __if self.head__.

Reply:
Yeah. Thanks for pointing out. 
Made the change: 65069374b7460b9cb809643510bde838e0e60970